src/main/java/org/prebid/server/bidder/adhese/AdheseBidder.java

```diff
@@ -145,6 +149,13 @@ private static String getRefererParameter(Site site) {
                 : "";
     }
 
+    private static String getIfaParameter(Device device) {
+        final String ifa = device != null ? device.getIfa() : null;
+        return StringUtils.isNotBlank(ifa)
+                ? String.format("%s%s", QUERY_PARAMETER_IFA, ifa)
+                : "";
```

Comment:
Minor. Maybe one line will be better
` return device != null && StringUtils.isNotBlank(device.getIfa())
                ? String.format("%s%s", QUERY_PARAMETER_IFA, device.getIfa())
                : "";`